.travis.yml

```diff
@@ -17,85 +12,76 @@ before_install:
 install:
   -
 
-env:
-  -JDK=Java8
-  -JDK=Java9
-  -JDK=Java10
-  -JDK=Java11-EA
-  -JDK=Java12-EA
-
 matrix:
   fast_finish: true
   include:
     - jdk: oraclejdk8
       env:
         - DESC="acceptance tests"
-        - JDK=Java8
         - CMD="mvn install --projects acceptance-tests --also-make --activate-profiles all"
 
     - jdk: oraclejdk8
       env:
         - DESC="findbugs"
-        - JDK=Java8
         - CMD="mvn install findbugs:check --projects '!scala-unit-tests,!jmh-scala-tests,!jmh-tests,!jcstress-tests,!p2-repository' --activate-profiles all -DskipTests=true"
+
     - jdk: oraclejdk8
       env:
         - DESC="checkstyle"
-        - JDK=Java8
         - CMD="mvn install checkstyle:check --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk8
       env:
         - DESC="unit tests"
-        - JDK=Java8
         - CMD="mvn install"
 
     - jdk: oraclejdk8
       env:
         - DESC="compile performance-tests"
-        - JDK=Java8
         - CMD="mvn install --projects performance-tests --also-make --activate-profiles all -DskipTests=true"
 
     - jdk: oraclejdk9
       env:
-        - DESC="unit tests Java 9"
-        - JDK=Java9
+        - DESC="unit tests oraclejdk9"
         - CMD="mvn install"
```

Comment:
Removing as many JDK declarations as possible, and changing the DESCriptions to use the Travis name of the jdk in as many places as possible, for consistency.

Reply:
is oraclejdk9 needed in the DESC if it is mentioned in the jdk?

Reply:
I guess not. It's just for clarity and appearance on this page:
https://travis-ci.org/eclipse/eclipse-collections

WDYT?

Reply:
Makes sense. I thought it was some connection between the mentioned jdk and DESC. 👍